src/commandlineflags.cc

```diff
@@ -175,7 +167,7 @@ bool ParseBoolFlag(const char* str, const char* flag, bool* value) {
   if (value_str == nullptr) return false;
 
   // Converts the string value to a bool.
-  *value = !(*value_str == '0' || *value_str == 'f' || *value_str == 'F');
+  *value = value_str[0] == '\0' || IsTruthyFlagValue(value_str);
```

Comment:
This now returns `true` for the empty string. Simply calling `IsTruthyFlagValue` should give you the right answer.